consensus/replay_test.go

```diff
@@ -150,7 +150,7 @@ LOOP:
 		t.Logf("====== LOOP %d\n", i)
 
 		// create consensus state from a clean slate
-		logger := log.NewNopLogger()
+		// logger := log.NewNopLogger()s
```

Comment:
Probably a good idea to set a logger, since it's possible to get `nil` panics otherwise - but in any case, I think we should either remove it completely or just keep it as-is.